app/cypress/integration/navigation.spec.js

```diff
@@ -0,0 +1,269 @@
+context("navigation", () => {
+  beforeEach(() => {
+    cy.visit("./public/index.html#/projects");
+  });
+
+  context("when the app is opened for the first time", () => {
+    it("shows the projects screen", () => {
+      cy.contains("Projects").should("exist");
+    });
+
+    it("shows the four default mock projects", () => {
+      cy.contains("monokel").should("exist");
+      cy.contains("Monadic").should("exist");
+      cy.contains("open source coin").should("exist");
+      cy.contains("radicle").should("exist");
+    });
+  });
+
+  context("first-level sidebar", () => {
+    it("provides navigation to all main sections of the app", () => {
+      cy.get('[data-cy="sidebar"]')
+        .get('a[title="Search"]')
+        .click()
+        .blur();
+      cy.contains("Search").should("exist");
+
+      // check whether the active icon is highlighted
+      cy.get('[data-cy="sidebar"]').toMatchImageSnapshot({
+        name: "sidebar-search-active"
+      });
+
+      cy.get('[data-cy="sidebar"]')
+        .get('a[title="Feed"]')
+        .click()
+        .blur();
+      cy.contains("Feed").should("exist");
+
+      // check whether the active icon is highlighted
+      cy.get('[data-cy="sidebar"]').toMatchImageSnapshot({
+        name: "sidebar-feed-active"
+      });
+
+      cy.get('[data-cy="sidebar"]')
+        .get('a[title="Projects"]')
+        .click()
+        .blur();
+      cy.contains("Projects").should("exist");
+
+      // check whether the active icon is highlighted
+      cy.get('[data-cy="sidebar"]').toMatchImageSnapshot({
+        name: "sidebar-projects-active"
+      });
+
+      cy.get('[data-cy="sidebar"]')
+        .get('a[title="Create new project"]')
+        .click()
+        .blur();
+      cy.contains("Design System").should("exist");
```

Comment:
This could be rewritten as
```suggestion
      cy.get("[data-cy=some-value]").should("contain", "Design System");
```
This allows for more consistent and strict assertions.